client/components/gravatar/index.jsx

```diff
@@ -46,22 +38,23 @@ export class Gravatar extends Component {
 		size: 32,
 	};
 
+	state = { failedToLoad: false };
+
 	getResizedImageURL( imageURL ) {
 		const { imgSize } = this.props;
 		imageURL = imageURL || 'https://www.gravatar.com/avatar/0';
-		const parsedURL = url.parse( imageURL );
-		const query = parse( parsedURL.query );
+		const parsedURL = getUrlParts( imageURL );
```

Comment:
fun fact, in these cases we can also perform our "deletion" in a destructuring assignment

```js
const { search, ...parsedURL } = getUrlParts( imageURL );
```

Just provides one way to remove one more thing to keep track of (what properties on `parsedURL` remain and which have been or may have been deleted).

Reply:
Good idea, thanks! I thought the linter would complain about `search` not being used anywhere, but that doesn't appear to be the case 👍